common/api-review/performance-exp.api.md

```diff
@@ -13,10 +13,7 @@ export interface FirebasePerformance {
 }
 
 // @public
-export function getPerformance(app: FirebaseApp): FirebasePerformance;
-
-// @public
-export function initializePerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;
+export function getPerformance(app: FirebaseApp, settings?: PerformanceSettings): FirebasePerformance;

```

Comment:
seems unrelated to this PR. and it's overwriting a newer version of the api report

Reply:
Reverted.